lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -1,181 +1,303 @@
 import os
 import shutil
 import subprocess as sp
+from pathlib import Path
 
 import numpy as np
 import pandas as pd
 import pkg_resources
 import pytest
 
-from lstchain.io.io import dl1_params_lstcam_key
+from lstchain.io.io import dl1_params_lstcam_key, dl2_params_lstcam_key
 from lstchain.io.io import dl1_params_src_dep_lstcam_key
-from lstchain.tests.test_lstchain import test_dir, mc_gamma_testfile, produce_fake_dl1_proton_file, fake_dl1_proton_file
-
-output_dir = os.path.join(test_dir, 'scripts')
-dl1_file = os.path.join(output_dir, 'dl1_gamma_test_large.h5')
-merged_dl1_file = os.path.join(output_dir, 'script_merged_dl1.h5')
-dl2_file = os.path.join(output_dir, 'dl2_gamma_test_large.h5')
-file_model_energy = os.path.join(output_dir, 'reg_energy.sav')
-file_model_disp = os.path.join(output_dir, 'reg_disp_vector.sav')
-file_model_gh_sep = os.path.join(output_dir, 'cls_gh.sav')
+from lstchain.tests.test_lstchain import (
+    test_dir,
+    mc_gamma_testfile,
+    produce_fake_dl1_proton_file,
+    fake_dl1_proton_file,
+    test_drive_report,
+    test_drs4_pedestal_path,
+    test_calib_path,
+    test_time_calib_path,
+    test_r0_path
+)
+
+output_dir = Path(test_dir, "scripts")
+dl1_file = output_dir / "dl1_gamma_test_large.h5"
+merged_dl1_file = output_dir / "script_merged_dl1.h5"
+dl2_file = output_dir / "dl2_gamma_test_large.h5"
+file_model_energy = output_dir / "reg_energy.sav"
+file_model_disp = output_dir / "reg_disp_vector.sav"
+file_model_gh_sep = output_dir / "cls_gh.sav"
+# Real data files to be produced in the tests
+real_data_dl1_file = output_dir / ('dl1_' + test_r0_path.with_suffix('').stem + '.h5')
+real_data_dl2_file = output_dir / ('dl2_' + test_r0_path.with_suffix('').stem + '.h5')
+# FIXME: improve the muons file path naming
+# muons_file = output_dir / ('muons_' + test_r0_path.stem)  # This does not work since the stream is stripped out
+muons_file = output_dir / 'muons_LST-1.Run02008.0000_first50.fits'
+datacheck_file = output_dir / ('datacheck_dl1_' + test_r0_path.with_suffix('').stem + '.h5')
 
 
 def find_entry_points(package_name):
-    '''from: https://stackoverflow.com/a/47383763/3838691'''
+    """from: https://stackoverflow.com/a/47383763/3838691"""
     entrypoints = [
         ep.name
-        for ep in pkg_resources.iter_entry_points('console_scripts')
+        for ep in pkg_resources.iter_entry_points("console_scripts")
         if ep.module_name.startswith(package_name)
     ]
     return entrypoints
 
 
-ALL_SCRIPTS = find_entry_points('lstchain')
+ALL_SCRIPTS = find_entry_points("lstchain")
 
 
 def run_program(*args):
```

Comment:
can we add an option `cwd` here so we can run them in the `tmp_path`?

Reply:
Is this `cwd` option still needed? Or it is just enough with the temporal directory fixtures.

Reply:
I think it's fine like it is.